lib/ansible/parsing/quoting.py

```diff
@@ -25,7 +25,19 @@ def is_quoted(data):
 
 
 def unquote(data):
-    ''' removes first and last quotes from a string, if the string starts and ends with the same quotes '''
+    """Removes first and last quotes from a string, if the string starts and
+    ends with the same quotes.
+
+    @param data: The string to modify
+    @type data:  str
+
+    @return: The string without quotes
+    @rtype : str
```

Comment:
We use sphinx docstring format, see https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
It's fix, thanks.